drivers/st25r3911b/Kconfig

```diff
@@ -0,0 +1,42 @@
+#
+# Copyright (c) 2019 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+menuconfig ST25R3911B_LIB
+	bool "NFC ST25R3911B library"
+	help
+	  Enable the NFC ST25R3911B library
+
+if ST25R3911B_LIB
+
+config ST25R3911B_SPI_FREQ
+	int "ST25R3911B SPI frequency"
+	default 4000000
+	help
+	  ST25R3911B SPI frequency
+
+config ST25R3911B_SPI_PORT
+	int "ST25R3911 SPI port"
```

Comment:
Calling it "SPI port" is confusing. Especially when right below you've got "IRQ port" (which is actually the IRQ line GPIO port). It should be rather called "SPI instance". I know, most of these settings will be eventually moved to DT, once it is possible to add dts bindings in nrf repository. But maybe it would be good to improve the naming now.
It's not a blocker from my side though.